tests/logging_helper_test.py

```diff
@@ -202,6 +183,25 @@ def test_zipkin_handler_raises_exception_if_ann_and_bann_not_provided(
             " for foo span" == str(excinfo.value))
 
 
+@mock.patch('pyramid_zipkin.logging_helper.thrift_obj_in_bytes', autospec=True)
+def test_log_span(thrift_obj):
+    # Not much logic here, so this is basically a smoke test
+    thrift_obj.return_value = 'obj'
+    registry = {'zipkin.transport_handler': (lambda x, y: (x, y))}
+    stream, span_bytes = logging_helper.log_span(
+        span_id='0x2',
```

Comment:
@kaisen these fake IDs will change to be "01d3120f591212" (16 char hex strings) in response to your PR.